gradle/prepareCogboardCompose.gradle.kts

```diff
@@ -18,6 +18,16 @@ services:
       - COGBOARD_VERSION=$currentVersion
     volumes:
       - "./mnt:/data"
+  
+  mongo:
+    image: mongo
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: "root"
+      MONGO_INITDB_ROOT_PASSWORD: "root"
```

Comment:
czy te rzeczy nie powinny być sparametryzowane?

Reply:
O ile dobrze rozumiem, to jest skrypt generujący plik Compose. Czy więc ten wygenerowany plik nie jest miejscem na parametryzację tych ustawień?

Reply:
local docker compose służy tylko do lokalnego developmentu. Ten skrypt, jak rozumiem powinien zbudować compose'a dla zadanej konfiguracji projektowej, więc przypuszczam, że tutaj powinniśmy jednak mieć parametry. Możemy się wstrzymać do jutra i dowiem się, co Szymon na to